### PR TITLE
Move workspaces under packages

### PR DESCRIPTION
**Pull request by joshwooding**
This PR moves tooling playground and docs under packages/ and moves the packages already there into a design-system folder.

This is so we can get rid of some modular patches

### CODE REVIEW

**Review by origami-z — APPROVED** [review]

**Review by origami-z — DISMISSED** [review]

File: packages/design-system/ag-grid-theme/scripts/build.js [hunk 1]
```diff
@@ -24,17 +22,23 @@ function buildStyles(entry) {
         // `node_modules`.
         findFileUrl(url) {
           if (!url.startsWith("~")) return null;
+
+          const nonPrefixedUrl = url.substring(1);
+          const resolvedFile = require.resolve(
+            nonPrefixedUrl.substring(0, nonPrefixedUrl.indexOf("/"))
```
Comment (joshwooding) [inline packages/design-system/ag-grid-theme/scripts/build.js:28]: cc @ararus @origami-z 

I edited the build script slightly to be less sensitive to position of node_modules.

File: .yarn/patches/modular-scripts-npm-3.1.0-5a349ac39e.patch [hunk 2]
```diff
@@ -77,20 +61,6 @@ index 4f6555e3988e6a229a64e2c429107e0f62526d39..d8ef7201d651d92262f1cc422f1136a4
...
-index 5587b7cda8a2841b4fd7c2950127cc3a102f3f19..905456cb8e02b32b4f105da274a3d4d72ddca440 100644
---- a/react-scripts/config/paths.js
-+++ b/react-scripts/config/paths.js
-@@ -88,6 +88,9 @@ module.exports = {
-   appSrc: resolveApp('src'),
-   modularSrc: [
-     resolveModular('packages'),
-+	resolveModular('playground'),
-+	resolveModular('tooling'),
-+	resolveModular('docs'),
```
Comment (EleanorHaproff) [inline .yarn/patches/modular-scripts-npm-3.1.0-5a349ac39e.patch:90]: No `resolveModular('design-tools')`?

**Review by EleanorHaproff — COMMENTED** [review]
So there will now be 4 packages: design-system, docs, playground and tooling? That makes sense to me 👍 
For my own curiosity, which modular patches are you trying to get rid of and why?

File: .yarn/patches/modular-scripts-npm-3.1.0-5a349ac39e.patch (hunk 2 above)
Comment (joshwooding) [reply .yarn/patches/modular-scripts-npm-3.1.0-5a349ac39e.patch:90]: Since it's under the packages workspace it's not needed.

**Comment by joshwooding** [comment]
> So there will now be 4 packages: design-system, docs, playground and tooling? That makes sense to me 👍 For my own curiosity, which modular patches are you trying to get rid of and why?

So those aren't really packages just workspaces under the "packages" folder. The packages are still core, lab etc.

**Comment by joshwooding** [comment]
Closing for now

**Pull request closed without merging**